ansible/rke2/airgap/inventory/group_vars/all.yml.template

```diff
@@ -0,0 +1,92 @@
+---
+# Global variables for RKE2 airgap deployment with tarball installation
+
+# RKE2 Configuration
+rke2_version: "v1.31.11+rke2r1"
+rke2_server_options: |
+  cluster-cidr: {{ cluster_cidr }}
+  service-cidr: {{ service_cidr }}
+  cluster-dns: {{ cluster_dns }}
+  disable:
+    - rke2-snapshot-controller
+    - rke2-snapshot-controller-crd
+    - rke2-snapshot-validation-webhook
+rke2_agent_options: ""
+
+# Installation Method Configuration
+# Available methods:
+#   - "tarball": Uses pre-downloaded RKE2 tarballs with embedded images (no registry required)
+#     Playbook: rke2-tarball-playbook.yml
+#     Best for: Pure airgap environments, simple deployments
```

Comment:
I don't think we need to include these comments since they are mentioned multiple times in the README as well